aspnetcore/blazor/get-started.md

```diff
@@ -15,91 +15,115 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 
-   # [Visual Studio for Mac](#tab/visual-studio-mac)
+Blazor Server is supported in Visual Studio for Mac. Blazor WebAssembly isn't supported at this time. To build Blazor WebAssembly apps on macOS, follow the guidance on the **.NET Core CLI** tab.
 
-   1\. Install [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/).
+1. Install [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/).
 
-   2\. Select **File** > **New Solution** or create a **New Project**.
+1. Select **File** > **New Solution** or create a **New Project**.
 
-   3\. In the sidebar, select **.NET Core** > **App**.
+1. In the sidebar, select **.NET Core** > **App**.
 
-   4\. Select the **Blazor Server App** template. Only the Blazor Server template is available in Visual Studio for Mac at this time. For a Blazor WebAssembly experience, follow the instructions on the **.NET Core CLI** tab. After selecting the Blazor Server template, select **Next**. For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>.
+1. Select the **Blazor Server App** template. Select **Create**.
 
-   <!-- For a Blazor WebAssembly experience, select the **Blazor WebAssembly App** template. Select **Next**. -->
+   For information on the Blazor Server hosting model, see <xref:blazor/hosting-models>.
 
-   5\. Set the **Target Framework** to **.NET Core 3.1** and select **Next**.
+1. Set the **Target Framework** to **.NET Core 3.1** and select **Next**.
 
-   6\. In the **Project Name** field, name the app `WebApplication1`. Select **Create**.
+1. In the **Project Name** field, name the app `WebApplication1`. Select **Create**.
 
-   7\. Select **Run** > **Run Without Debugging** to run the app *without the debugger*. Run the app with **Start Debugging** to run the app *with the debugger*.
+1. Select **Run** > **Run Without Debugging** to run the app *without the debugger*. Run the app with **Start Debugging** to run the app *with the debugger*.
 
-   If a prompt appears to trust the development certificate, trust the certificate and continue.
+If a prompt appears to trust the development certificate, trust the certificate and continue.
 
-   # [.NET Core CLI](#tab/netcore-cli/)
+# [.NET Core CLI](#tab/netcore-cli/)
 
-   For a Blazor WebAssembly experience, execute the following commands in a command shell:
+1. Install the [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
+
+1. Optionally install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template:
+
+   * Install the [.NET Core 3.1.201 or later (Preview) SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
+   * Run the following command in a command shell. The [Microsoft.AspNetCore.Components.WebAssembly.Templates](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.Templates/) package has a preview version while Blazor WebAssembly is in preview.
```

Comment:
Again, I think we can drop these sub bullets.

Reply:
Already done.